eks-log-collector.sh

```diff
@@ -363,8 +363,13 @@ get_ipamd_info() {
 
 get_sysctls_info() {
   try "collect sysctls information"
-  
+  # dump all sysctls
   sysctl --all >> "${COLLECT_DIR}"/sysctls/sysctl_all.txt 2>/dev/null
+  
+  # dump relevant sysctls
```

Comment:
Does not the `sysctl --all` do the same thing?  Also "for f" should be more descriptive.

Reply:
You are correct. `sysctl --all` does that same. Removing as this redundant.